dockerfile/rocm4.0-pytorch1.7.0.dockerfile

```diff
@@ -72,8 +72,9 @@ ENV PATH="${PATH}" \
 
 WORKDIR ${SB_HOME}
 
+ARG ROCM_ARCH=gfx908
 ADD third_party third_party
-RUN ROCM_VERSION=rocm-4.0.0 make -j -C third_party rocm
+RUN ROCM_VERSION=rocm-4.0.0 ROCM_ARCH=${ROCM_ARCH} make -j -C third_party rocm
```

Comment:
why not build for all amdgpu targets?

Reply:
it's a long history that mi100 machine can not successfully build rocblas in this rocm4.0_pytorch1.7 docker, and and guys provide some extra restrict on building including limit gpu target, otherwise rocblas can not build successfully